scripts/clang-format/clang-format.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+cd ${PWD}/scripts/clang-format # assume current dir is arangodb (not error-proof)
```

Comment:
I think instead of `${PWD}/scripts/clang-format` we should use `"${PWD}/scripts/clang-format"` here (plus of all following occurrences of variables). Otherwise it may not work with directory names that contain spaces.